buildkitd/dockerd-wrapper.sh

```diff
@@ -35,7 +35,15 @@ execute() {
                 # Exiting here on the last retry maintains prior behavior of exiting when this cant start.
                 exit 1
             fi
-            sleep 5
+
+            if grep -q "^failed to start containerd: timeout waiting for containerd to start$" /var/log/docker.log; then
```

Comment:
do we need to clear `/var/log/docker.log` between runs, so that when we grep it, we're only looking at output from the most recent run? maybe we need a `> /var/log/docker.log` here.

Reply:
I checked with a test script and the log is overwritten with our current syntax.